src/PrestaShopBundle/Resources/config/services/core/form/choice_provider.yml

```diff
@@ -1,6 +1,8 @@
 services:
   _defaults:
     public: true
+    bind:
+      $translator: '@translator'
```

Comment:
Hum, i agree with this, but why adding it here, as you can add autowire: true in the new service definitions ?

Reply:
I did add autowire true in new definitions, that required me to bind this translator argument. 
I don't get it whats wrong here :smile:

Reply:
It can be injected without adding it in the configuration if you type hint with the TranslatorInterface, what you did is absolutely not wrong, but may not be needed with types :D

Reply:
ok finally I autowired the TranslatorInterface itself, so its not needed to bind that argument anywhere anymore. Didn't check other usages, but we can clean them up at some point